librz/io/p/io_srec.c

```diff
@@ -0,0 +1,802 @@
+// Copyright: Manolache183 <manolache.alexandru8@gmail.com>
+
+/*
+*** SREC format description : every line follows this pattern
+	S / Type / Bytecount / Address / Data / Checksum
+
+	There are a bunch of types, from which one (S4) is reserved
+
+//sauce : https://en.wikipedia.org/wiki/SREC_(file_format)
+
+**** example records
+
+S00F000068656C6C6F202020202000003C
+S11F00007C0802A6900100049421FFF07C6C1B787C8C23783C6000003863000026
+S11F001C4BFFFFE5398000007D83637880010014382100107C0803A64E800020E9
+S5030003F9
+S9030000FC
+*/
+
+#include "rz_io.h"
```

Comment:
Please use angular braces here. Also there is no need for `<stdio.h>`, `<stdlib.h>`, and `<sys/types.h>`. Just use `<rz_types.h>` instead.